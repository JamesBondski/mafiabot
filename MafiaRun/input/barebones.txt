G:bots 8
0:join #mafia2
0:say #mafia2 ##create test barebones
A:join #mafia_test
0:say #mafia_test ##start
0:say #mafia_test ##voice
1:say #mafia_test ##vote Bot1
2:say #mafia_test ##vote Bot1
3:say #mafia_test ##vote Bot1
4:say #mafia_test ##vote Bot1
0:say #mafia_test ##kill Bot2
0:say #mafia_test ##voice