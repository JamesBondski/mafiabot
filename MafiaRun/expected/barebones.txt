Joined-#mafia2:#mafia2
Joined-#mafia_test:#mafia_test
C-#mafia2:Game created! To join the game, go to channel #mafia_test.
Op-#mafia_test:Bot0
C-#mafia_test:Bot1 has joined the game (1 players now). To leave the game, leave the channel or use the ##leave command.
C-#mafia_test:Bot2 has joined the game (2 players now). To leave the game, leave the channel or use the ##leave command.
C-#mafia_test:Bot3 has joined the game (3 players now). To leave the game, leave the channel or use the ##leave command.
C-#mafia_test:Bot4 has joined the game (4 players now). To leave the game, leave the channel or use the ##leave command.
C-#mafia_test:Bot5 has joined the game (5 players now). To leave the game, leave the channel or use the ##leave command.
C-#mafia_test:Bot6 has joined the game (6 players now). To leave the game, leave the channel or use the ##leave command.
C-#mafia_test:Bot7 has joined the game (7 players now). To leave the game, leave the channel or use the ##leave command.
Mode-#mafia_test:+m
C-#mafia_test:The game has started. You can talk as soon as the mod uses the ##voice command.
Mode-#mafia_test:+vvvvvvv Bot1 Bot2 Bot3 Bot4 Bot5 Bot6 Bot7
C-#mafia_test:Count: Bot1(Bot1).
C-#mafia_test:Count: Bot1(Bot1, Bot2).
C-#mafia_test:Count: Bot1(Bot1, Bot2, Bot3).
Devoice-#mafia_test:Bot1
C-#mafia_test:Bot1 has been killed.
Mode-#mafia_test:-vvvvvvv Bot1 Bot2 Bot3 Bot4 Bot5 Bot6 Bot7
C-#mafia_test:Count: Bot1(Bot1, Bot2, Bot3, Bot4).
Devoice-#mafia_test:Bot2
Mode-#mafia_test:+vvvvvvv Bot3 Bot4 Bot5 Bot6 Bot7
